Makefile

```diff
@@ -179,6 +179,9 @@ notice:
 		-noticeTemplate dev-tools/notice/NOTICE.txt.tmpl \
 		-noticeOut NOTICE.txt \
 		-depsOut ""
+	@if [ $(shell go env GOOS) == "windows" ]; then \
+		cat x-pack/packetbeat/npcap/installer/LICENSE >> NOTICE.txt; \
+	fi
```

Comment:
I'm not at all sure that this is a sensible idea and that it's not better to unconditionally append the text with words to indicate that it is only for x-pack packetbeat.